BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -96,15 +96,48 @@ public struct AccessibilityElement: Element {
 
 extension Element {
 
-    /// Wraps the element to provide the passed accessibility
-    /// options to the accessibility system.
+
+    /// Wraps the receiver in an accessibility element with the provided values.
+    ///
+    /// - Important: ⚠️ This overrides the accessibility of the contained element and all of its children ⚠️
+    ///
+    /// - SeeAlso: ``accessibilityElement``
```

Comment:
Installing Xcode 13 to ensure this doc link works as expected, but happy to use a different reference syntax if that's preferred.

Reply:
+1 for writing new docs in docc format; we'll be supporting it soon!